tests/helper.py

```diff
@@ -0,0 +1,39 @@
+import dpctl
+import dpnp
+
+
+def get_all_dtypes(no_bool=False,
+                   no_float16=True,
+                   no_complex=False,
+                   no_none=False,
+                   device=None):
+    """
+    Build a list of types supported by DPNP based on input flags and device capabilities.
+    """
+
+    dev = dpctl.SyclQueue().sycl_device if device is None else device
```

Comment:
```suggestion
    dev = dpctl.select_default_device() if device is None else device
```

Reply:
The `SyclQueue().sycl_device` does a lot more work, also creating a context and structures for scheduler.